src/components/RepeatableActionsField/RepeatableActionsField.js

```diff
@@ -36,6 +36,7 @@ export const RepeatableActionsField = memo(({
     <Headline
       tag="h3"
       margin="xx-small"
+      data-test-repeatable-field-legend
```

Comment:
Consider placing data-test- attribute on top to maintain uniformity in the module.